Frameworks/CoreText/CTParagraphStyle.mm

```diff
@@ -17,235 +17,182 @@
-            },
-
-        [kCTParagraphStyleSpecifierFirstLineHeadIndent] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _firstLineHeadIndent),
-            },
-        [kCTParagraphStyleSpecifierHeadIndent] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _headIndent),
-            },
-        [kCTParagraphStyleSpecifierTailIndent] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _tailIndent),
-            },
-        [kCTParagraphStyleSpecifierDefaultTabInterval] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _tabInterval),
-            },
-        [kCTParagraphStyleSpecifierLineBreakMode] =
-            {
-                sizeof(CTLineBreakMode), offsetof(_CTParagraphStyleProperties, _lineBreakMode),
-            },
-        [kCTParagraphStyleSpecifierLineHeightMultiple] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _lineHeightMultiple),
-            },
-        [kCTParagraphStyleSpecifierMaximumLineHeight] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _maximumLineHeight),
-            },
-        [kCTParagraphStyleSpecifierMinimumLineHeight] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _minimumLineHeight),
-            },
-        [kCTParagraphStyleSpecifierLineSpacing] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _lineSpacing),
-            },
-        [kCTParagraphStyleSpecifierParagraphSpacing] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _paragraphSpacing),
-            },
-        [kCTParagraphStyleSpecifierParagraphSpacingBefore] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _paragraphSpacingBefore),
-            },
-        [kCTParagraphStyleSpecifierBaseWritingDirection] =
-            {
-                sizeof(CTWritingDirection), offsetof(_CTParagraphStyleProperties, _writingDirection),
-            },
-        [kCTParagraphStyleSpecifierMaximumLineSpacing] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _maximumLineSpacing),
-            },
-        [kCTParagraphStyleSpecifierMinimumLineSpacing] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _minimumLineSpacing),
-            },
-        [kCTParagraphStyleSpecifierLineSpacingAdjustment] =
-            {
-                sizeof(CGFloat), offsetof(_CTParagraphStyleProperties, _lineSpacingAdjustment),
-            },
+#pragma region __CTParagraphStyle
+typedef struct __CTParagraphStyle : CoreFoundation::CppBase<__CTParagraphStyle> {
+    __CTParagraphStyle() {
+        static CFArrayRef __defaultValues = []() {
```

Comment:
>CFArrayRef [](start = 15, length = 10)

perhaps this should be a `static unique_cf`, just to round it all out. That way when the module unloads, *CFRelease* will be called.